src/components/structures/RoomView.js

```diff
@@ -360,6 +360,10 @@ module.exports = React.createClass({
                 });
             }
         }
+
+        // update ther tab complete list as it depends on who most recently spoke,
+        // and that has probably just changed
+        this._updateTabCompleteList();
```

Comment:
This should probably be moved up so it is inside:
```javascript
if (ev.getSender() !== MatrixClientPeg.get().credentials.userId) {
  // ...
}
```
I don't think there is any point in updating the tab-completion ordering with your own user.